lib/routes/mangadex/_feed.ts

```diff
@@ -0,0 +1,181 @@
+    const includes = needCover ? ['cover_art'] : [];
+
+    const rawMangaMetas = (await cache.tryGet(
+        `mangadex:manga-meta:${md5(deDuplidatedIds.join(''))}`, // shorten the key
+        async () => {
+            const { data } = await got.get(
+                constants.API.MANGA_META.slice(0, -1) +
+                    toQueryString({
+                        ids: deDuplidatedIds,
+                        includes,
+                        limit: deDuplidatedIds.length,
+                    })
+            );
+
+            if (data.result === 'error') {
+                throw new Error('Failed to retrieve manga meta from MangaDex API.');
+            }
+            return data.data;
+        }
+    )) as any;
+
+    const languages = [...(typeof lang === 'string' ? [lang] : lang || []), ...(await getFilteredLanguages())].filter(Boolean);
+
+    return rawMangaMetas.reduce((map, rawMangaMeta) => {
+        const id = rawMangaMeta.id;
+
+        const titles = {
+            ...rawMangaMeta.attributes.title,
+            ...Object.fromEntries(rawMangaMeta.attributes.altTitles.flatMap((element) => Object.entries(element))),
+        };
+
+        const title = firstMatch(titles, [...languages, rawMangaMeta.attributes.originalLanguage]) as string;
+
+        const description = firstMatch(rawMangaMeta.attributes.description, languages) as string;
+
+        let cover: string | undefined;
+        let manga = { id, title, description, cover };
+
+        if (needCover) {
+            const coverFilename = rawMangaMeta.relationships.find((relationship) => relationship.type === 'cover_art')?.attributes.fileName + '.512.jpg';
+            cover = `${constants.API.COVER_IMAGE}${rawMangaMeta.id}/${coverFilename}`;
+            manga = { ...manga, cover };
+        }
+
+        map.set(id, manga);
+        return map;
+    }, new Map<string, { id: string; title: string; description: string; cover?: string }>());
+}
+
+/**
+ * Retrieves the chapters of a manga.
+ *
+ * @author chrisis58, vzz64
+ * @param id manga id
+ * @param lang language(s), absent for default
+ * @returns chapters of the manga
+ */
+const getMangaChapters = async (id: string, lang?: string | string[], limit?: number) => {
+    const languages = new Set([...(typeof lang === 'string' ? [lang] : lang || []), ...(await getFilteredLanguages())].filter(Boolean));
+
+    const url = `${constants.API.MANGA_META}${id}/feed${toQueryString({
+        order: {
+            publishAt: 'desc',
+        },
+        limit: limit || 100,
+        translatedLanguage: languages,
+    })}`;
+
+    const chapters = (await cache.tryGet(
+        `mangadex:manga-chapters:${id}`,
+        async () => {
+            const { data } = await got.get(url);
+
+            if (data.result === 'error') {
+                throw new Error(data.errors[0].detail);
+            }
+
+            return data.data;
+        },
+        config.cache.contentExpire,
```

Comment:
https://github.com/DIYgod/RSSHub/pull/18269#discussion_r2072422509
```suggestion
        config.cache.routeExpire,
```
This belongs to a hot cache which requires a shorter TTL.